### ISSUE TITLE
Support assigning numeric value to "editor.wrappingIndent"

### ISSUE THREAD

**Issue by minacle** [Microsoft/vscode#42203]
- VSCode Version: 1.19.3

`editor.wrappingIndent` is currently supporting `"none"`, `"same"`, `"indent"`.

The problem with `"same"` is that it is difficult to know where the new line starts, and the problem with `"indent"` is that it is difficult to know where the new block starts. (Also, I don't like `"none"` 😜)

Many IDEs and editors support custom numeric value for wrapping indent.
I hope this feature is also supported in Visual Studio Code.

**Comment by alexreg** [comment]
+1 for this

**Comment by Spleeding1** [comment]
Yes absolutely. I would like a 1.5 indent setting. So if indent is set to 4, wrap is 6. If indent is 2, wrap is 1. etc.

```python
class MyClass(object):
    def method(variable_1, variable_2, variable_3
      variable_4): # wrapped at 1.5 indent
        # method code
```


```javascript
function myFunction() {
  document.querySelectorAll('nav > ul >
   li:first-child').forEach(function(){
    // function code
  });
}
```
